### PR TITLE
fix: calling data from backend

### PR DESCRIPTION
**Pull request by yokwejuste**
In relation to issue #119 This PR was raised.

### CODE REVIEW

**Review by Zaker237 — APPROVED** [review]

**Comment by Asam237** [comment]
@yokwejuste 
Screenshot from 2022-08-10 08-17-16

**Review by Sanix-Darker — CHANGES_REQUESTED** [review]
Did you tested this pr ?

**Comment by yokwejuste** [comment]
> Did you tested this pr ?

Yes Sanix! The PR where individually tested and running well.

File: frontend/src/pages/streetMap/StreetMap.page.tsx [hunk 1]
```diff
@@ -11,7 +11,6 @@ import {
     Marker,
     useMapEvents,
 } from "react-leaflet";
-import { LIST_VILLE } from "../../scripts/list_ville";
 import { LIST_QUARTIER } from "../../scripts/list_quartier";
```
Comment (Asam237) [inline frontend/src/pages/streetMap/StreetMap.page.tsx:14]: Why does this file still exist?

File: frontend/src/pages/streetMap/StreetMap.page.tsx [hunk 2]
```diff
@@ -85,7 +83,8 @@ const StreetMap = (props: any) => {
             >
                 <MyComponent />
                 <TileLayer
-                    attribution='&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> &copy; <a href="http://cartodb.com/attributions">CartoDB</a>'
+                    attribution='&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> &copy;
+                    <a href="http://cartodb.com/attributions">CartoDB</a>'
                     url="https://cartodb-basemaps-{s}.global.ssl.fastly.net/dark_all/{z}/{x}/{y}.png"
                 />
                 {LIST_QUARTIER.map((item: any, index: any) => (
```
Comment (Asam237) [inline frontend/src/pages/streetMap/StreetMap.page.tsx:90]: Remove all constant files

**Review by Asam237 — CHANGES_REQUESTED** [review]

File: frontend/src/pages/streetMap/StreetMap.page.tsx (hunk 1 above)
Comment (yokwejuste) [reply frontend/src/pages/streetMap/StreetMap.page.tsx:14]: Please! An issue surely with the management of my branches. Le tme make an update

**Review by Asam237 — APPROVED** [review]

**Pull request merged**